idioms-json-2.0/campaign-v-actors.json

```diff
@@ -52,6 +52,33 @@
             "target_ref": "identity--2d1c6ab3-5e4e-48ac-a32b-f0c01c2836a8",
             "type": "relationship"
         },
+        {
+            "created": "2014-08-08T15:50:10.983Z",
+            "id": "relationship--d145dbd0-42bf-4684-8b5b-74be315d8e27",
+            "modified": "2014-08-08T15:50:10.983Z",
+            "relationship_type": "attributed-to",
+            "source_ref": "incident--229ab6ba-0eb2-415b-bdf2-079e6b42f51e",
```

Comment:
Should we be seen these on a STIX 2.0 run? If yes, the incidents are missing from this bundle (all relationships have IDs to different Incidents).